src/bin/tpcds_table_generator.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+
+#include "storage/storage_manager.hpp"
+#include "tpcds/tpcds_table_generator.hpp"
+
+int main() {
+  std::cout << "TPCDS" << std::endl;
+  std::cout << " > Generating tables" << std::endl;
+
+  opossum::TpcdsTableGenerator(1, 1000).generate_and_store();
```

Comment:
The second parameter of the constructor is the chunk size, right? Why `1000`?

Reply:
There was a tpch_table_generator executable, which used chunk size = 1000. I just copied the code and adjusted it for tpcds.

Reply:
There was a tpch_table_generator which used chunk size = 1000. I just copied the code and adjusted it for tpcds.